test/unit/app/models/offsite_link_test.rb

```diff
@@ -147,4 +147,23 @@ class OffsiteLinkTest < ActiveSupport::TestCase
       end
     end
   end
+
+  test "creating an existing offsite link republishes the parent" do
+    test_object = create(:world_location_news, world_location: create(:world_location))
```

Comment:
Would this be better as a more descriptive `parent` variable?

Reply:
Yes, that seems like a reasonable change (I just copied this test from somewhere else).